src/Transport/Configuration/SettingsKeys.cs

```diff
@@ -13,5 +13,6 @@ static class SettingsKeys
         public const string RuleNameShortener = Base + nameof(RuleNameShortener);
         public const string TransportType = Base + nameof(TransportType);
         public const string CustomTokenProvider = Base + nameof(CustomTokenProvider);
+        public const string Self = Base + "1BA47B92-6987-4B9C-AFF3-B906B9FED0B3";
```

Comment:
`Self` - could you please provide a more meaningful name? Without reading the [comment](https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/blob/f33a5b9929a6bd663c8eeaac7359fb6529956607/src/Transport/Receiving/TransactionScopeSuppressFeature.cs) it's hard to determine what this key is for.

Reply:
Feel free to change it to your needs. We couldn't come up with a better name. Sorry